save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ExecutionView.kt

```diff
@@ -34,14 +39,44 @@ external interface ExecutionProps : RProps {
     var executionId: String
 }
 
+/**
+ * A state of execution view
+ */
+external interface ExecutionState : RState {
+    /**
+     * Execution dto
+     */
+    var executionDto: ExecutionDto?
+}
+
 /**
  * A [RComponent] for execution view
  */
 @OptIn(ExperimentalJsExport::class)
 @JsExport
-class ExecutionView : RComponent<ExecutionProps, RState>() {
+class ExecutionView : RComponent<ExecutionProps, ExecutionState>() {
+    init {
+        state.executionDto = null
+    }
+
+    override fun componentDidMount() {
+        GlobalScope.launch {
+            val headers = Headers().also { it.set("Accept", "application/json") }
+            get("http://localhost:5000/getDto?executionId=${props.executionId}", headers)
+                .json()
+                .await()
+                .unsafeCast<ExecutionDto?>()
+                ?.let {
+                    setState { executionDto = it }
+                }
+        }
+    }
+
     @Suppress("EMPTY_BLOCK_STRUCTURE_ERROR", "TOO_LONG_FUNCTION")
     override fun RBuilder.render() {
+        div {
+            +("Project version: ${(state.executionDto?.version ?: "None")}")
```

Comment:
```suggestion
            +("Project version: ${(state.executionDto?.version ?: "N/A")}")
```
`N/A` or `undefined` I think would be more clear